### DISCUSSION TITLE
Hyprland Window Manager?

### DISCUSSION THREAD

**Discussion by greysonofusa** [clearlinux/distribution#2884] [Ideas]
I was wondering if we could add Hyprland as a compositor to the official repo.

https://hyprland.org/

It's quite impressive!

**Comment by fenrus75** [comment]
I understand the ask but for us right now, each now "compositor" ends up as a whole environment (with all the testing/bugload/etc to go with it)... so it's really about "would this REPLACE sway"  .. since adding one more is a lot less attractive to us.
